doc/conf.py

```diff
@@ -13,10 +13,14 @@
 import os
 import sys
 
-from pyflagser import __version__
-
+# This line should be called before importing pyflagser
```

Comment:
Interesting!  (I'm not sure I understand.)

Reply:
Previously, in order to generate the documentation, you need to install the package in the system to retrieve the information of the version. 
When doing this, you don't need to install the package in order to generate the documentation.
I don't know if I made myself clear

Reply:
Perfectly clear, thanks!